server.js

```diff
@@ -76,38 +76,29 @@ app.get('/api/profile', function profileIndex(req, res) {
   })
 });
 
-app.get('/api/restaurant', function restaurantIndex(req, res){
+app.get('/api/restaurants', function restaurantIndex(req, res){
   db.Restaurant.find({}, function(err, allRestaurant){
-  res.json(allRestaurant) 
+    res.json(allRestaurant) 
   })
 });
 
-app.post('/api/restaurant', function restaurantCreat(req, res){
+app.post('/api/restaurants', function restaurantCreat(req, res){
   db.Restaurant.create(req.body, function(err, newRestaurant){
   if (err) { console.log('error', err); }
   res.json(newRestaurant);
   });
 });
 
 
-// app.get('/api/restaurant/:id', function restaurantShow(req, res){
```

Comment:
As a general rule, please get rid of commented out code (i.e. dead code) before you commit.